mldsa/fips202/fips202.c

```diff
@@ -475,7 +475,7 @@ __contract__(
  *              Modifies the state. Can be called multiple times to keep
  *              squeezing, i.e., is incremental.
  *
- * Arguments:   - uint8_t *out: pointer to output
+ * Arguments:   - uint8_t *out: pointer to mutable output data
```

Comment:
Nit: I prefer the original formulation. Calling `out` out as 'mutable' suggests that the data stored at `out` at call-time is being operated on, but that's not the case: It's just overwritten.

Reply:
I see the point. I guess the word "mutable" does distinguish between "write only" and "read/write". I will remove it.